Update anchor-links.md (#3041)
enkidevs/curriculum: web/html/links/anchor-links.md

## Revision

> Add the missing code to create an anchor link that navigates to the portion of the page whose id is `contact`:

Reviewer: what do you think about this?

```suggestion
Create an anchor link that navigates to the HTML element with the `contact` id:
```

Author: Sounds good
